docs/core/stories.rst

```diff
@@ -49,13 +52,73 @@ This is what we call a **story**.
   in the format ``intent{"entity1": "value", "entity2": "value"}``.
 - Actions executed by the bot are shown as lines starting with ``-``
   and contain the name of the action.
-- Events returned by an action are on lines immediately after that
-  action. For example, if an action returns a ``SetSlot`` event,
-  this is shown as the line ``- slot{"slot_name": "value"}``
+- Events returned by an action are on lines immediately after that action.
+  For example, if an action returns a ``SlotSet`` event, this is shown as
+  ``slot{"slot_name": "value"}``.
+
+Let's now take a slightly more detailed look at each of these components,
+along with some things you should keep in mind while using them to write
+your own stories.
+
+User Messages
+~~~~~~~~~~~~~
+While writing stories, you do not have to deal with the specific contents of
+the messages that the users send. Instead, you can take advantage of the output
+from the NLU pipeline, which lets you use just the combination of an intent and
+entities to refer to all the possible messages the users can send to mean the
+same thing.
+
+It is important to include the entities here as well because the policies learn
+to predict the next action based on a *combination* of both the intent and
+entities (you can, of course, customize this behavior using the
+:ref:`use_entities <use_entities>` attribute).
+
+Actions
+~~~~~~~
+While writing stories, you will encounter two types of actions: utterances
+and custom actions. Utterances are hardcoded messages that a bot can respond
+with. Custom actions, on the other hand, involve custom code being executed. 
+
+All actions (both utterances and custom actions) executed by the bot are shown
+as lines starting with ``-`` followed by the name of the action. For custom
+actions, the action name is the string you choose to return from the ``name``
+method of the custom action class. All utterances must begin with the prefix
+``utter_``, and custom actions must begin with ``action_``.
```

Comment:
```suggestion
``utter_``.
```
I wish 😃 it's just a convention we use that it seems like barely anyone else uses